manifests/init.pp

```diff
@@ -19,17 +21,14 @@
     fail('cgroups::package_name must be a string or an array.')
   }
 
+  validate_hash($mounts)
+  validate_hash($groups)
+
+
   case $::osfamily {
     'RedHat': {
-      case $::operatingsystemmajrelease {
-        '6': {
-          $default_package_name   = 'libcgroup'
-          $default_cgconfig_mount = '/cgroup'
-        }
-        default: {
```

Comment:
That would add support for EL7, since its values are the same and allow us to keep the pattern of failing by default for unknown platforms.

The real goal is that we fail by default if the platform is not explicitly supported